Apps/WebClient/src/Server/Services/UserProfileService.cs

```diff
@@ -109,7 +109,7 @@ public RequestResult<UserProfileModel> GetUserProfile(string hdid, DateTime jwtA
                 return new RequestResult<UserProfileModel>()
                 {
                     ResultStatus = ResultType.Success,
-                    ResourcePayload = new UserProfileModel(),
+                    ResourcePayload = new UserProfileModel() { IdentityProviderEmail = jwtEmailAddress },
```

Comment:
this is odd. Why is it necessary to set the jwt email to the one passed? If there are no profiles on the backend the object is empty indicating that

Reply:
This is where IdentityProviderEmail is populated.  `jwtEmailAddress` is fetched in GetUserProfile() in the controller.